packages/devtools_app/lib/src/shared/side_panel.dart

```diff
@@ -129,6 +140,13 @@ class SidePanelViewerState extends State<SidePanelViewer>
       ),
     );
   }
+
+  @override
+  void dispose() {
+    cancelListeners();
```

Comment:
we can delete this call to `cancelListeners`. Since this class mixes in AutoDisposeMixin, this will happen automatically.